go/exorelay/bin/lint

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 set -e
 
+go install
```

Comment:
Can you please add this as a comment to this line in all files? Future generations of developers who don't see this PR might wonder about this as well  :)